summingbird-online/src/main/scala/com/twitter/summingbird/online/executor/FutureQueue.scala

```diff
@@ -141,6 +188,11 @@ private[summingbird] class FutureQueue[S, T](
     }
   }
 
+  /**
+   * Retrieve any completed results, along with their associated states.
+   *
+   * Returns up to maxEmitPerExec items.
+   */
   def dequeue: TraversableOnce[(S, Try[T])] = {
```

Comment:
Yeah I guess `Queue` is set to return `Seq`; seems reasonable to do the same here.